src/sonic-host-services/scripts/hostcfgd

```diff
@@ -406,6 +512,9 @@ class HostConfigDaemon:
         self.kdumpCfg = KdumpCfg(self.config_db)
         self.kdumpCfg.load(self.config_db.get_table('KDUMP'))
 
```

Comment:
@arlakshm 
we need to do self.masicBgpMonCfg = None . 
SIngle asic is getting exception.

Please update.

Reply:
@arlakshm updated in latest commit. Please check.

Reply:
@abdosi, lgtm.